execution_workflows/DaPars2/Dapars2_snakemake/snakemake.yaml

```diff
@@ -0,0 +1,110 @@
+name: snakemake
```

Comment:
Is this the unchanged `docs/templates/snakemake/snakemake.yaml` ? If yes, then we do not need this and it can be deleted.